public/app/plugins/datasource/grafana-azure-monitor-datasource/components/VariableEditor/VariableEditor.tsx

```diff
@@ -286,6 +288,26 @@ const VariableEditor = (props: Props) => {
           />
         </InlineField>
       )}
+      {query.queryType === AzureQueryType.AzureResourceGraph && (
+        <>
+          <ArgQueryEditor
+            subscriptionId={query.subscription}
```

Comment:
not sure this one is needed? 

It is my understanding that ARG relies on `subscriptions` (plurals) instead?
But I was also wondering if leaving it here helps with the default subscription? Since the singular form is used in `ArgQueryEditor`:
https://github.com/grafana/grafana/blob/9c6aab3bc950ea6bd1c65ecf0f50f8cfe43ede94/public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ArgQueryEditor/ArgQueryEditor.tsx#L48

Any insight appreciated 😃  (I've tested both locally and they both work the same way, but I don't know how edge cases would be handled)

Reply:
I think it makes sense to leave this in 😊

Reply:
yes, in case a default subscription is configured it should use this one, but there is a corner case :)

If a user select other variable type, for example "Resource Groups" and change the subscription, that value will be saved in the `query.subscription`, which will be later used here for the ARG editor, even though that input is no longer in the screen. I believe a better value for this would be `datasource.azureLogAnalyticsDatasource.defaultSubscriptionId` which is always pointing to the subscription.